tests/integration/test_wu.py

```diff
@@ -205,25 +198,20 @@ class UserStatusGivenCorrectly(WuAtlantisFakeDBInitialized):
     def setUp(self):
         super().setUp()
         self.valid_status_nutzer_list = NutzerFactory.create_batch(20)
-        self.unknown_status_nutzer_list = UnknownStatusNutzerFactory.create_batch(20)
+        self.unknown_status_nutzer_list = UnknownStatusNutzerFactory\
```

Comment:
Please revert this useless difference. The line length is not too important here.